en/08_Changelogs/5.4.0.md

```diff
@@ -32,6 +33,20 @@ We have provided a severity rating of the vulnerabilities below based on the CVS
 
 ## Features and enhancements
 
+### Logged warning if allowed hosts have not been configured {#allowed-hosts-warning}
+
+If [`Director.alternate_base_url`](api:SilverStripe\Control\Director->alternate_base_url) is not set, the "host" header is used by Silverstripe CMS to determine what the host name is for your project. This is useful when creating absolute URLs, e.g. for use in emails.
+
+Notably this is used in [`Director::host()`](api:SilverStripe\Control\Director::host()), which in turn is called by many methods including [`Director::hostName()`](api:SilverStripe\Control\Director::hostName()), [`Director::protocolAndHost()`](api:SilverStripe\Control\Director::protocolAndHost()), [`Director::is_site_url()`](api:SilverStripe\Control\Director::is_site_url()), and [`Director::absoluteURL()`](api:SilverStripe\Control\Director::absoluteURL()).
+
+Ideally your hosting will reject invalid host headers. For example Apache allows you to define valid hosts as part of the virtual host configuration, and a Web Application Firewall (WAF) can also be configured to validate the host header. However if your hosting is set to allow *any* host header, your project might be vulnerable to host header injection attacks.
+
+You can configure Silverstripe CMS to validate the host header, which is an extra layer of protection against this type of attack. While you should have appropriate validation at a hosting level, it is best practice to also configure this in your project.
+
+This configuration has existed since 2016, but we've been alerted that many projects still have not configured their hosting nor their project to adequately validate host headers. To help prompt developers, we've added a warning which will be logged if the configuration is not set.
```

Comment:
> To help prompt developers, we've added a warning which will be logged if the configuration is not set.

This bit needs a lot more prominence, since it is the actual change.  If should basically be in its own paragraph and say "If you haven't configured SS_ALLOWED_HOSTS, or set some of the new YML (which you should either add an example of in this change log, or link direclty to an example in docs)", then yours logs will now fill up with warnings.

Reply:
It is already it's own paragraph? I've added _another_ paragraph afterward with wording similar to the language you've suggested.

> or  set some of the new YML (which you should either add an example of in this change log, or link direclty to an example in docs)

The YAML isn't knew. There is a link to the example in the docs - that's the last line in this section. We usually only add code examples for _new_ API, but this API isn't new by any means.